src/config/config.json

```diff
@@ -1,4 +1,5 @@
 {
   "PORT": 3000,
-  "MONGO_URI": "mongodb://127.0.0.1:27017/items"
-}
\ No newline at end of file
+  "MONGO_URI": "mongodb://admin:admin@mongo:27017/?authSource=admin",
+  "MONGO_URI_TEST": "mongodb://127.0.0.1:27017/items"
```

Comment:
There should be one mongo uri and different databases should be handled on test setup. 

I would rather split those to different config values a.k.a MONGODB_CONNECTION, MONGODB_PASSWORD, MONGODB_DATABASE and use them in the mongoose connect params.

Reply:
e34ccd39407d34189ac7db0e271bda287ee63182
fixed